collector/src/main/java/com/bitmovin/analytics/BitmovinAnalytics.java

```diff
@@ -49,15 +55,15 @@ public BitmovinAnalytics(BitmovinAnalyticsConfig bitmovinAnalyticsConfig, Contex
         this.bitmovinAnalyticsConfig = bitmovinAnalyticsConfig;
         this.playerStateMachine = new PlayerStateMachine(this.bitmovinAnalyticsConfig, this);
         this.playerStateMachine.addListener(this);
-        this.eventDataDispatcher = new SimpleEventDataDispatcher(this.bitmovinAnalyticsConfig, this.context, this);
+        this.eventDataDispatcher = new SimpleEventDataDispatcher(this.bitmovinAnalyticsConfig, this.context, this, debugCallback);
```

Comment:
I like the idea but IMO the debugCallback should not be part of the SimpleEventDataDispatcher but rather a decorator to be more SRP